Katello/katello
# Fixes #35410 - Fix a pathing issue on exports

#### What are the changes introduced in this pull request?
When exporting a content view version with a space its name we include that space in the generated path. Its not necessarily a bug but ideally we'd like path to be something like -> 
`/var/lib/pulp/exports/Test_Organization/Test_CV/1.0/2022-08-03T00-17-11-00-00/metadata.json`
vs 
`/var/lib/pulp/exports/Test_Organization/Test CV/1.0/2022-08-03T00-17-11-00-00/metadata.json`

The cv name without and with space.

#### Considerations taken when implementing this change?
This issue was introduced when the previous syncable export work got merged and got caught on bats tests. So useful.

#### What are the testing steps for this pull request?
- Create a CV name with a space in it
- Add a simple custom repo or rh repo with `immediate` download policy
- Publish
- Export the content view version via hammer. 
 - `hammer content-export complete version --id=<cvv-id>`
 
Before this PR, you 'd see something like

`/var/lib/pulp/exports/Test_Organization/Test CV/1.0/2022-08-03T00-17-11-00-00/metadata.json` with the space

After this PR

`/var/lib/pulp/exports/Test_Organization/Test_CV/1.0/2022-08-03T00-17-11-00-00/metadata.json` with the space
